src/video_core/renderer_opengl/gl_texture_cache.cpp

```diff
@@ -112,13 +112,17 @@ GLenum ImageTarget(Shader::TextureType type, int num_samples = 1) {
     return GL_NONE;
 }
 
-GLenum TextureMode(PixelFormat format, bool is_first) {
+GLenum TextureMode(PixelFormat format, std::span<SwizzleSource> swizzle) {
```

Comment:
Passing this small 4 byte array by copy is likely to be faster than passing a span
```suggestion
GLenum TextureMode(PixelFormat format, std::array<SwizzleSource, 4> swizzle) {
```

Reply:
Should I just pass it by reference?